Kinvey.Core/Auth/User.cs

```diff
@@ -703,17 +703,18 @@ static public async Task GetMICAccessTokenAsync(String token, string micID = nul
 			}
 		}
 
-		#endregion
+        #endregion
 
-		#region Realtime
+        #region Realtime
 
-		/// <summary>
-		/// Registers the active user for realtime messaging.
-		/// </summary>
-		/// <returns>The realtime.</returns>
-		/// <param name="userClient">User client.</param>
-		/// <param name="ct">Ct.</param>
-		public async Task RegisterRealtimeAsync(AbstractClient userClient = null, CancellationToken ct = default(CancellationToken))
+        /// <summary>
+        /// Registers the active user for realtime messaging.
+        /// </summary>
+        /// <returns>The realtime.</returns>
+        /// <param name="userClient">User client.</param>
+        /// <param name="ct">Ct.</param>
+        /// <param name="realtimeReconnectionPolicy"> Realtime reconnection policy </param>
+        public async Task RegisterRealtimeAsync(AbstractClient userClient = null, CancellationToken ct = default(CancellationToken), RealtimeReconnectionPolicy realtimeReconnectionPolicy = RealtimeReconnectionPolicy.None)
```

Comment:
I'm questioning myself if we should set `RealtimeReconnectionPolicy.None` as the default value
It seems that PubNub decided to change their `Reconnection Policy` from `v3` and `v4`. `NONE` is now the default in `v4`

https://www.pubnub.com/docs/c-sharp-dot-net-c-sharp/migration-guide-from-v3

what do you guys think guys? @vinaygahlawat @romanogolikhin

Reply:
@romanogolikhin is it possible to reconnect manually?

Reply:
@vinaygahlawat @heyzooi 
About default value. It seems to me this is up to us. If we decide, we can change default value.
I set default value in "None" according to v4. 

Yes. It is possible to reconnect manually.
https://www.pubnub.com/docs/c-sharp-dot-net-c-sharp/pubnub-network-lifecycle#reconnection-manually
But unfortunately this is not going to work. @Vinay sent message to PubNub support with information about troubles. Waiting for answer.

Reply:
IMO we should change the default value to be `EXPONENTIAL`, other than that LGTM